wordpress-mobile/WordPress-Android
# Magic Link login

Fixes #3959 

To test:

1. Sign in with email you know is WordPress.com email and login by requesting email
2. Sign in with email you know is WordPress.com email and login with password
3. Sign in with username and verify password request is shown and is able to log in
4. Click 'Add self-hosted site` and verify it works

Needs review: @maxme
